java/vertx/pom.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>benchmark.vertx</groupId>
+    <artifactId>benchmark</artifactId>
+    <version>1.0.0</version>
+    <packaging>jar</packaging>
+    <name>vertx4</name>
+    <description>Demo project for Vertx</description>
+
+    <properties>
+        <java.version>11</java.version>
+        <vertx.version>4.0.1</vertx.version>
```

Comment:
Sadly, you can't. The reason is that I'm using a bill of materials to import all versions in dependencies. (You will see that the dependencies have no version). It is either lock them all here, or, remove the BOM and add versions manually to the dependencies. I'm not picky so if you really want a range I can rework the `pom.xml`.

Reply:
The decision is yours 😛 I think we rather stick to `vertx` usage, not introduce new ones ...

In fact, using range was a decision taken to accomodate this project maintenance (and since this appear to be a standard usage with `maven`).

Since, I am a `rubyist`, I can confirm that it is a common practice to use some kind of ranges, but I can not speak for other languages